src/drc/api/serializers.py

```diff
@@ -198,6 +221,9 @@ def create(self, validated_data):
             informatieobjecttype = self._get_informatieobjecttype(validated_data['informatieobjecttype'])
             validated_data['vertrouwelijkheidaanduiding'] = informatieobjecttype['vertrouwelijkheidaanduiding']
 
+        canonical = EnkelvoudigInformatieObjectCanonical.objects.create()
```

Comment:
we're no longer operating on just a single instance that gets or doesn't get created, so _probably_ we should explicitly wrap `def create` in `@transaction.atomic`